src/CMakeLists.txt

```diff
@@ -1009,9 +1030,9 @@ install(TARGETS osdmaptool DESTINATION bin)
 
 set(ceph_psim_srcs
   tools/psim.cc)
-add_executable(ceph-psim ${ceph_psim_srcs})
-target_link_libraries(ceph-psim global)
-install(TARGETS ceph-psim DESTINATION bin)
+add_executable(ceph_psim ${ceph_psim_srcs})
```

Comment:
it does make sense. but i thought we should limit the scope of this PR to cmake only changes. will add a commit to change the packaging and automake also.